source/cli.rst

```diff
@@ -86,6 +138,16 @@ Description
 
 Debug GLPI cache.
 
+Arguments/Options
+****************
+
+There are no arguments for this command
+
+Options:
+
+- ``-k``, ``--key`` (required) - Cache key to debug. (no default)
```

Comment:
This one can be used multiple times (`InputOption::VALUE_IS_ARRAY`), it should be mentionned.